modules/core/src/parallel.cpp

```diff
@@ -77,6 +86,10 @@
     #define HAVE_CONCURRENCY
 #endif
 
+#if defined __linux__ || defined __GLIBC__ || defined __HAIKU__ || defined __EMSCRIPTEN__
+   #define CV_HAVE_CGROUP
```

Comment:
`__EMSCRIPTEN__` doesn't have cgroups.

Reply:
Can you suggest another name :) ? I think it's easier if we include  __EMSCRIPTEN__